public/emojis.js

```diff
@@ -84,57 +97,73 @@ const emojis = {
   CHAIR: "🪑",
   CHART: "📈",
   CHECK: "✅",
+  CHEERS: "🥂",
   CHEESE: "🧀",
   CHEF: "👩‍🍳",
   CHERRY: "🍒",
   CHESS: "♟️",
   CHICK: "🐥",
   CHILD: "👶",
+  CHOP: "🔪",
   CIRCLE: "⭕",
   CIRCUS: "🎪",
   CLAMP: "🗜️",
   CLAP: "👏",
+  CLIMB: "🧗‍♀️",
   CLIP: "📎",
   CLOUD: "☁️",
   CLOVER: "🍀",
   CLOWN: "🤡",
   CLUB: "♣️",
   COAT: "🧥",
+  COCO: "🥥",
+  COCOA: "🍫",
+  CODER: "👩‍💻",
   COFFIN: "⚰️",
   COIN: "🪙",
+  COLOR: "",
```

Comment:
???

Reply:
thanks for catching this - the build didn't fail 😱 

I'll add a test